spec/enumerize/integrations/rspec/matcher_spec.rb

```diff
@@ -221,7 +221,7 @@ def self.name
   describe '#with_multiple' do
 
     it 'accepts when has defined the multiple' do
-      model.enumerize(:sex, :in => [:male, :female], multiple: true)
+      model.enumerize(:sex, :in => [:male, :female], multiple: true, default: :male)
```

Comment:
This test looks legit and I think that we shouldn't change it. I think that [the matcher itself](https://github.com/brainspec/enumerize/blob/master/lib/enumerize/integrations/rspec/matcher.rb) needs to be changed to work for cases when `default` is not set.